e2e/scripts/check-e2e-wallet-balance.ts

```diff
@@ -8,4 +8,8 @@ import { checkBalance, getCeloTokensBalance } from './utils'
   console.log(`E2E_TEST_WALLET_SECURE_SEND: ${E2E_TEST_WALLET_SECURE_SEND}`)
   console.table(await getCeloTokensBalance(E2E_TEST_WALLET_SECURE_SEND))
   await checkBalance(E2E_TEST_WALLET_SECURE_SEND)
+
+  console.log(`E2E_TEST_FACUET: ${E2E_TEST_FAUCET}`)
```

Comment:
Ah I see that this is in the ticket AC... I think my point above still applies, but perhaps we can just make sure that the faucet itself has a good buffer of funds. Either way, I suppose this just shifts left a bit on when the e2e tests start to fail, since they'll eventually fail anyways if the faucet is low and we attempt to fund the wallets.

Reply:
Changed to report the balance of the faucet and not fail in 0881c28aa06c88449bb33c33fd4cbfa4128086c4.